go.mod

```diff
@@ -0,0 +1,8 @@
+module sigs.kubernetes.io/sig-windows-tools
+
+go 1.14
+
+require (
+	github.com/Masterminds/semver v1.5.0
```

Comment:
this should work for us for now, but i haven't tried it for the k8s specific metadata (if we decide to build kube-proxy from master one day).

i know the apimachinery/pkg/util/version works for sure.

Reply:
Yeah, if we need something we can change later